docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -360,6 +338,49 @@ public FormValidation doCheckNumExecutors(@QueryParameter int numExecutors) {
             return FormValidation.ok();
         }
 
+        public FormValidation doCheckVolumesString(@QueryParameter String volumesString) {
```

Comment:
Better such validation rather then no at all. This variant is already 10 times rewritten locally :laughing: